BuildProject/App.tsx

```diff
@@ -6,6 +6,7 @@
  */
 
 import PromotedMetrics, { ActionType } from '@promotedai/react-native-metrics';
+import type { AncestorIds } from '@promotedai/react-native-metrics';
```

Comment:
is there an autoformatter for rn?

Reply:
Yeah, I can add a lint target to the package.json.